frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -74,13 +83,25 @@ const applicationsUserAdministers = computed(() => {
 
 const tabViewRef = ref();
 
-onUnmounted(() => {
+const resetNewTag = () => {
+    router.push({ query: {} });
+    newAppAdminId.value = '';
+    newUserAccessIds.value = '';
+    newDelegatedAdminIds.value = '';
+};
+
+const resetNotificationAndNewRowTag = () => {
     resetNotification();
+    resetNewTag();
+};
```

Comment:
This seems to be the main control of reset, maybe just call it `reset`? Just wondering...